packages/typescript/mixin.core.js

```diff
@@ -1,10 +1,22 @@
 const { existsSync } = require('fs');
 const { join } = require('path');
 const { Mixin } = require('hops-mixin');
+const chalk = require('chalk');
+
+const hasTypescriptInstalled = packageJsonPath => {
+  const { dependencies, devDependencies = {} } = require(packageJsonPath);
+  const { typescript } = { ...dependencies, ...devDependencies };
+
+  return typeof typescript === 'string';
+};
 
 class TypescriptMixin extends Mixin {
-  configureBuild(webpackConfig, { jsLoaderConfig, allLoaderConfigs }) {
+  configureBuild(webpackConfig, { jsLoaderConfig, allLoaderConfigs }, target) {
     const { loader, options, exclude } = jsLoaderConfig;
+    const targetDevelop = target === 'develop';
```

Comment:
Should be skip the webpack configuration if we know that we don't have the typescript dependency installed?
```js
if (!hasTypescriptInstalled(packageJsonPath)) {
  return;
}
```

Reply:
Good point, forgot about that.

Reply:
@jhiode & I agreed that applying this build configuration conditionally makes no sense in the context of `hops-typescript` being the conscious decision of a user to have & use Typescript in their project.